templates/lookup/ifupdown__combined_interfaces.j2

```diff
@@ -67,7 +67,11 @@
 {%       endif %}
 {%     endif %}
 {%     if params.allow is undefined %}
-{%       set _ = params.update({"allow":["hotplug"]}) %}
+{%       if (ansible_virtualization_role == 'guest' and ansible_virtualization_type in [ 'lxc', 'openvz' ]) %}
+{%         set _ = params.update({"allow":["boot"]}) %}
```

Comment:
:+1: for this fix. I just came across this shorty before trying this PR. I can confirm that this fixes the issue I had with network not brought up on boot in LXC containers.